src/test/java/bio/terra/service/snapshot/SnapshotTest.java

```diff
@@ -179,6 +178,9 @@ public void snapshotUnauthorizedPermissionsTest() throws Exception {
         assertThat("Dataset filters to dataSnapshots",
             enumSnapByBadDatasetId.getTotal(),
             equalTo(0));
+
+        // Delete snapshot as custodian for this test since teardown uses steward
+        dataRepoFixtures.deleteSnapshot(custodian(), snapshotSummary.getId());
```

Comment:
I saw this in a few spots around the tests. I'm confused as to why custodians would have permission to delete a snapshot. From the permissions spreadsheet, custodians only have permissions to manipulate datasets.

Reply:
It's a little weird but notice that for this particular case, we happen to create the snapshot with the user labeled custodian.  By creating the snapshot, that user is actually a steward for that snapshot